tsb/fqdn/aws/variables.tf

```diff
@@ -0,0 +1,18 @@
+variable "dns_zone" {
+    default = null
+}
+variable "fqdn" {
+}
+variable "address" {
+}
+
+variable "tsb_mp" {
```

Comment:
Same question than before, we shouldn't be needing this. What am I missing?

Reply:
not need at this stage, so can be taken down - but more a consistency matter for the further expansion based on the infra state, @nauticalmike is prepping ocp module (to deploy ocp at gcp, azure, etc) so we may be of use... however you are right, it's solely optional and not needed at this stage

Reply:
I think it is better to not introduce unused variables for the sake of consistency. Whilst I see what you're trying to achieve, I think it brings, now, more confusion than value. For the new reader of the repo (like me or any customer looking at it), it just makes it harder to understand, because you have to figure out what is really needed and what is just noise for the sake of developer convenience.

In my opinion, we shouldn't introduce things until they're needed. Consistency is fine, but where appropriate, and IMHO, in this cases, it makes understanding the whole stuff harder.